augur/frequencies.py

```diff
@@ -26,8 +28,10 @@ def register_parser(parent_subparsers):
                         help="delimiters to accept when reading a metadata file. Only one delimiter will be inferred.")
     parser.add_argument('--metadata-id-columns', default=DEFAULT_ID_COLUMNS, nargs="+",
                         help="names of possible metadata columns containing identifier information, ordered by priority. Only one ID column will be inferred.")
-    parser.add_argument('--regions', type=str, nargs='+', default=['global'],
-                        help="region to subsample to")
+    parser.add_argument('--regions', type=str, nargs='+', default=[DEFAULT_REGION],
+                        help="region to subsample to. " \
+                            f"Regions should match values in the {REGION_COLUMN!r} column of the metadata file " \
```

Comment:
Based on recent thinking in the [subsampling](https://docs.google.com/document/d/10jrk4OaLhfo6io8WjtZuZFD9BDpr85GtghO1x5GAOM8/edit) doc, this behavior is filtering and not subsampling.


```suggestion
                        help="region to filter to. " \
                            f"Regions should match values in the {REGION_COLUMN!r} column of the metadata file " \
```

Reply:
Thanks, that is clearer! Updated in [force-push](https://github.com/nextstrain/augur/compare/20d9e21b11016cbbbdf274081669ebfbf9e26cb1..455bcfab3a46c41ed69de0d9921967ce63d11d1a)